website/blog/2022-05-31-aluminium.md

```diff
@@ -0,0 +1,294 @@
+
+ From this release we will also search the parent method in case the current method's scaladoc are empty. 
+
+## [Scala 3] Show scaladocs for signature help
+
+As mentioned in the previous paragraph, Metals can show you documentation in three different places. That, however, was true only for Scala 2 previously. In this release, we will now show you proper documentation whenever invoking signature help.
+
+As a reminder, signature help is used to indicate what parameters can be used in a method. It should pop up automatically after writing `(`, but you can also invoke it manually. In VS Code that takes the form of `editor.action.triggerParameterHints` command, which can also be bound to a shortcut and by default is.
+
+## [MUnit] Test Explorer can find helper methods from parent classes
+
+MUnit allows to use [helper functions](https://scalameta.org/munit/docs/tests.html#declare-tests-inside-a-helper-function) when declaring tests. Very often those helper methods are extracted to some parent classes which are extended by many test suites.
+Now, Metals can find usages of those helper methods and display them in Test Explorer.
+
+This feature is available for Bloop and SBT 1.7.0-M2 or later.
+
+![MUnit-helper-methods](https://imgur.com/GGRDpXA.gif)
+
+## Support Cats Effect stacktraces in stacktrace analyzer
+
+Cats Effect offers [asynchronous stack tracing](https://typelevel.org/cats-effect/docs/tracing#asynchronous-stack-tracing) which augments exceptions with additional information.  
+Now, Stacktrace Analyzer is able to recognize CE's stacktraces and provide link to location in code. Say no to tedious debugging when you have only stacktrace from the logs!
+
+![cats-effect-stacktraces](https://imgur.com/5fMvcYd.gif)
+
+
+## Improve implement all completion and code action
+Previously, when we invoked the "implement all members" completion and code action, Metals used `x$0` for Java parameter names. Now, Metals fills these with the correct parameter names.
+ 
+## Improve rewrite to braces/parenthesis code action
+Sometimes, it wasn't clear which code would be affected by [rewrite to braces/parenthesis](https://scalameta.org/metals/blog/2021/09/06/tungsten#replace--with--in-functions-and-vice-versa) code action.
+Now, code action's description contains name of the function/method which will be affected by executing action.
+![rewrite-braces-parens](https://imgur.com/SkHolsJ.gif)
+Theme: [One Dark Pro](https://marketplace.visualstudio.com/items?itemName=zhuangtongfa.Material-theme)
+
+
+## Support for Scala 3.1.3-RC4, 3.1.3-RC3
+Metals 0.11.6 supports Scala 3.1.3-RC3 and RC4.
+
```

Comment:
```suggestion
## Automatically setup java home for the Bloop build server
In previous versions of Metals if users wanted to change the java version of the Bloop build server they would need to manually update `.bloop/bloop.json` file in their user home directory. With this version we make sure that the Bloop Java version will correspond to Metals one in order to avoid weird compilation issues that could arise from different versions being used.

Now, each time you update `javaHome` or `bloopJvmProperties` settings Metals will now ask you whether to forward those changes to the Bloop configuration file. If you modified that file previously or want to use a custom one you can decide to dismiss the Metals notification. Otherwise, everything will be setup for you automatically once you decide to apply the changes. If you never created the file, we will create it for you before starting Bloop, so you should not notice anything out of order.
```

Reply:
included https://github.com/scalameta/metals/pull/3957/commits/dbea826c031425e31a2ad9e7a7fdbc11b3415405 !